src/fem/finite_element.hh

```diff
@@ -343,7 +352,7 @@ protected:
      * @param q Quadrature rule.
      * @param flags Update flags.
      */
-    virtual FEInternalData *initialize(const Quadrature<dim> &q);
+    FEInternalData *initialize(const Quadrature<dim> &q);
```

Comment:
Move this to FeValues together with FeInternalData.

Reply:
Done.